### PR TITLE
PPTP-435 Primary contact details check your answers page

### PR DESCRIPTION
**Pull request by PauloPortugal**
Screenshot from 2021-02-26 11-36-29

### CODE REVIEW

File: tampermonkey/PPT_AutoComplete.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 // ==UserScript==
 // @name         PPT AutoComplete
 // @namespace    http://tampermonkey.net/
-// @version      3.0
+// @version      4.0
```
Comment (SteveSugden) [inline tampermonkey/PPT_AutoComplete.js:4]: You will run out of versions at this rate!

**Review by SteveSugden — APPROVED** [review]
Seems sane

File: tampermonkey/PPT_AutoComplete.js (hunk 1 above)
Comment (PauloPortugal) [reply tampermonkey/PPT_AutoComplete.js:4]: Are you joking or should we follow a more industry standard approach? :smile:

Comment (SteveSugden) [reply tampermonkey/PPT_AutoComplete.js:4]: Was joking.  But we are going up in 0.1 increments on our scripts.  TM FTW!

**Pull request merged**